testpilot/experiments/models.py

```diff
@@ -11,7 +13,9 @@
 from ..utils import HashedUploadTo
 
 
+experiment_facebook_upload_to = HashedUploadTo('image')
```

Comment:
Yeah, I did it this way to prevent database errors in deployments. No new experiments will be allowed without these images (and old ones won't be able to be edited without them, for that matter), but old ones are allowed to exist without breaking constraints.